tools/ibc-solidity-abi/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+pub mod generated {
+    use ethers::prelude::abigen;
+
+    abigen!(MockTransfer, "src/abis/MockTransfer.json");
```

Comment:
If we only care about the `sendTransfer` method, we can use e.g.:

```rust
// Human readable ABI
abigen!(MockTransfer, r"[
    function sendTransfer(...)
]");
```

Or we can define an interface `ISendTransfer` having only a `sendTransfer` method and generate ABI for the interface.

Reply:
Done